tensorboard/webapp/metrics/views/right_pane/settings_view_component.scss

```diff
@@ -62,6 +62,12 @@ section .control-row:not(:last-child) {
   }
 }
 
+.scalars-smoothing .slider-input {
+  flex: none;
+  // Wide enough to show 3 digits after a decimal.
+  width: 60px;
```

Comment:
This is a bit surprising. Do we need it because of the flex container? If so, can we have a wrapper DOM to not hardcode a width like this? I think without this, it will have the browser default input width which may not be what you want either. In any case, maybe `5c` or something that can change with the font size like `5em` (not precisely correct but at least it is not a constant)?

Reply:
Changed to `em`s.

I was also surprised; even without flex, the problem still occurs.  A quick search shows others have filed bugs on behavior like this, so it seems to be a known browser incompatibility.
https://bugzilla.mozilla.org/show_bug.cgi?id=1242914